spec.html

```diff
@@ -28,14 +28,14 @@ <h1>Syntax</h1>
       <ins>`export` ExportFromClause FromClause IfClause `;`</ins>
 
     IfClause :
-      <ins>`if` `{` ConditionEntries `}` `;`</ins>
+      <ins>`if` `{` ConditionEntries `,`? `}`</ins>
```

Comment:
I assume the trailing semicolon is redundant because it exists in `ImportDeclaration`.

Reply:
Good catch!